app/scripts/lib/metrics.js

```diff
@@ -129,6 +133,8 @@ define(function (require, exports, module) {
     this._utmSource = options.utmSource || NOT_REPORTED_VALUE;
     this._utmTerm = options.utmTerm || NOT_REPORTED_VALUE;
     this._xhr = options.xhr || xhr;
+
+    this.initialize(options);
```

Comment:
This module already has an [init function](https://github.com/mozilla/fxa-content-server/pull/4718/files#diff-7b6eb0846bca35324d8b1acd38a32302R143) that is called from [app-start](https://github.com/mozilla/fxa-content-server/blob/313e69e7e3f8896ba60d37f3c6c48db6e6033710/app/scripts/lib/app-start.js#L208). Having both `init` and `initialize` is confusing. `init` is the odd one out compared to the rest of the repo.  Going along with the never settle theme, instead of introducing a similar sounding method that's called from a different place, can you rename `init` to `initialize` and remove the call to `metrics.init()` in app-start? Cocktail should take care of resolving the conflicting methods by calling them both.